tools/atip/atip/common/steps.py

```diff
@@ -59,6 +59,15 @@ def switch_to_app_name(context, app_name):
         assert False
 
 
+@step(u'quit "{app_name}"')
```

Comment:
My bad for not catching your comments before merged. Please drive a fixing separately, thanks.